clients/attune/attune_log_parser.py

```diff
@@ -0,0 +1,80 @@
+import os
+import csv
+import re
+
+os.getcwd()
+os.chdir('test_logs/')
+
+log_file = 'System_2020-11-10_18-52-23.log'
+#Define regexes
+
+User_dictionary = {
+    'test' : 'test1'
+}
```

Comment:
On how to load this user dictionary, the normal recommendation is to load 'secret' or external data either through environment variables or external files. I would tend to prefer external files for something like this. I would use a JSON file for this.

JSON data is very similar to the Python syntax, with some specific differences (like only double-quoted strings). After formatting your user dictionary as a separate JSON file (you can use websites like https://www.jsonformatter.io/)
```
{
   "some_long_uid": "cjohnsto",
   "some_other_uid": "abeitz"
}
```
then you can load this dictionary with:
```
import json

with open('test.json') as json_file:
  user_dictionary = json.load(json_file)
```

This also clearly separates the code from data. We can also just never commit the json file to source control, so it never appears publicly. This approach is currently being used on the server-side of labbot.